src/parse_data.py

```diff
@@ -0,0 +1,27 @@
+import json
+import logging
+from typing import Generator, Union, List, Dict, Any
+from datetime import datetime
+
+logger = logging.getLogger('parseStoriesInfo')
+
+def build_stories_info(stories_generator: Generator[dict, None, None]) -> str:
```

Comment:
Why do you need this parsing? Why not save it as is?

Reply:
This is necessary because I need to save data in a certain format
```
{
  "date": "[execution_date]",
  "articles": [
    {
      "rating": 1002,
      "name": "[name_of_the_article]",
      "link": "[absolute_link_to_the_article]"
    },
    ...
  ]
}
```